README.md

```diff
@@ -9,24 +9,28 @@ _Having problems? want to contribute? join our [community slack](http://devtools
 
 > Generate a CHANGELOG from git metadata
 
-It's recommended you use the high level [standard-version](https://github.com/conventional-changelog/standard-version) library, which handles the entire tagging and CHANGELOG
-generation process.
-
-Alternatively, the [conventional-changelog-cli](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-cli) allows you to generate a CHANGELOG from the
-command line.
-
 ## About this Repo
 
 The conventional-changelog repo is managed as a [monorepo](https://github.com/babel/babel/blob/master/doc/design/monorepo.md); it's composed of many npm packages.
 
 The original `conventional-changelog/conventional-changelog` API repo can be
 found in [packages/conventional-changelog](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog).
 
+## Get started
+
+It's recommended you use the high level [standard-version](https://github.com/conventional-changelog/standard-version) library, which handles the entire tagging and CHANGELOG
+generation process.
+
+Alternatively, the [conventional-changelog-cli](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-cli) allows you to generate a CHANGELOG from the command line.
```

Comment:
I think it might be worth considering deprecating `contentional-changelog-cli` unless anyone is actively keeping it up-to-date? I'd have a preference for us instead calling attention to [semantic-release](https://github.com/semantic-release/semantic-release). Perhaps:

```
It's recommended you use the high level [standard-version](https://github.com/conventional-changelog/standard-version) library, which is a drop-in replacement for npm's `version` command, handling automated version bumping, tagging and CHANGELOG generation.

Alternatively, if you'd like to move towards completely automating your release process as an output from CI/CD, consider using [semantic-release](https://github.com/semantic-release/semantic-release).
```

Reply:
Oh yeah. semantic-release was not updated for a long time when I was working on this so it wasn't added. We should add it now.